universal-login-relayer/lib/services/WalletService.ts

```diff
@@ -33,6 +36,17 @@ class WalletService {
     }
     throw new InvalidENSDomain(ensName);
   }
+
+  async deploy(key: string, ensName: string) {
+    const ensArgs = this.ensService.argsFor(ensName);
+    if (ensArgs !== null) {
+      const args = [key, ...ensArgs] as string[];
+      const initWithENS = encodeInitializeWithENSData(args);
+      const transaction = await this.factoryContract.createContract(key, initWithENS, {...defaultDeployOptions});
+      return transaction;
```

Comment:
`return await this.factoryContract.createContract(key, initWithENS, {...defaultDeployOptions});`